tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs

```diff
@@ -126,6 +126,31 @@ public async Task CanBuildAndRunAngular6App_WithDevAndProdDependencies_UsingZipp
                     var data = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/");
                     Assert.Contains("Angular6app", data);
                 });
+
+            // Re-run the runtime container multiple times against the same output to catch any issues.
```

Comment:
I think we should make a default in our test framework to re-run the runtime image multiple times against an output for ALL tests. This will simulate some scenarios of AppService and hopefull catch any issues.